bot_cli/commands.py

```diff
@@ -222,78 +207,34 @@ def show_all():
     show all exist contacts
     prints command result
     """
-    if book.data:
+    if address_book.data:
         result = list()
-        for record in book.data.values():
+        for record in address_book.data.values():
             result.append(str(record))
         print(Fore.LIGHTBLUE_EX + "\n".join(result))
     else:
         print(Fore.LIGHTBLUE_EX + "No contacts have been added yet")
 
 
-def birthdays():
-    get_birthdays_per_week = book.get_birthdays_per_week()
+@max_period_error
+def birthdays(args):
+    if args:
+        try:
+            period = int(args[0])
+            if (period < MIN_PERIOD) or (period > MAX_PERIOD):
+                raise ValueError
+        except:
+            raise ValueError
+    else:
+        period = DEFAULT_PERIOD
+
+    get_birthdays_per_week = address_book.get_birthdays_per_week(period)
```

Comment:
Maybe another name would be better here. Like `get_upcoming_birthdays`, because it's not for one week anymore